es6-classes/square.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+
+class Square extends Shape {
+  constructor(width) {
+    super(width);
+    this.width = width;
+    this.area = width * width;
+    this.perimeter = width * 4;
+  }
+
+  describe() {
+    return 'The perimeter is ' + this.perimeter + ', the area is ' +
+      this.area + ' , and the width is ' + this.width + '.';
+  }
```

Comment:
Instead of using `this.perimeter` & `this.area`, we want to utilize the `super` keyword. By calling `super.describe()` we can run the `describe` method that exists on the parent class.